### ISSUE TITLE
Postgres 15 support

### ISSUE THREAD

**Issue by sechmann** [nais/cli#274]
Postgres 15 removed default permissions from the public schema. This affects how cloudsqliam users get access to this schema. I think we create superusers for the applications themselves, so they and their credentials will still work.

**Comment by Reasonable-Solutions** [comment]
I looked into this and I think we can fix by using the `cloudsqliamuser` schema and granting privilieges on that. 
eg, diff like 
```
 
 var prepareDdlStatements = []string{
-	"alter default privileges in schema public grant CHANGEME on tables to cloudsqliamuser;",
-	"alter default privileges in schema public grant CHANGEME on sequences to cloudsqliamuser;",
-	"grant CHANGEME on all tables in schema public to cloudsqliamuser;",
-	"grant CHANGEME on all sequences in schema public to cloudsqliamuser;",
+	"alter default privileges in schema cloudsqliamuser grant CHANGEME on tables to cloudsqliamuser;",
+	"alter default privileges in schema cloudsqliamuser grant CHANGEME on sequences to cloudsqliamuser;",
+	"grant CHANGEME on all tables in schema cloudsqliamuser to cloudsqliamuser;",
+	"grant CHANGEME on all sequences in schema cloudsqliamuser to cloudsqliamuser;",
 }
```
 all users in postgres have a search path that looks like 
 `$user_name, public` so if a user needs a bunch of tables etc they can put them in the schema with the username

**Comment by mortenlj** [comment]
Fixed in #316
